tests/ui/conftest.py

```diff
@@ -77,6 +89,21 @@ def pytest_configure(config):
     prefix_indexes(config)
 
 
+@pytest.fixture
+def generate_addons(transactional_db):
+    from django.core.management import call_command
+    from django.utils import translation
+
+    from olympia.landfill.serializers import GenerateAddonsSerializer
+    
+    translation.activate('en-US')
+    serializer = GenerateAddonsSerializer()
+    serializer.create_installable_addon()
+    cache.clear()
+    call_command('clear_cache')
+
+
+"""
```

Comment:
why is that commented out?

Reply:
I think I removed something locally and it was failing.